docs/wiki/development/windows-provisioning.md

```diff
@@ -1,15 +1,26 @@
-# Windows 10 Provisioning Script
+# Windows 10 Development Environment Provisioning
 
-The PowerShell script `provision.ps1` is used to prepare a clean Windows 10 64 bit machine into one that is ready of **osquery** development on Windows. However, the script does _not_ automate the generation of the Visual Studio 2015 solution nor performs the build process.
+The bulk of the development environment provisioning logic is in the `tools\provision.ps1` PowerShell script. It is **not** recommended to directly invoke this script. Instead, run the `tools\make-win64-dev-env.bat` batch script. This provisioning script is used to prepare a clean Windows 10 64 bit operating system into one that is prepared for **osquery** development by downloading and installing the proper tools and dependencies.
+
+Generating the Visual Studio 2015 solution and building the **osquery** binaries is done via the `tools\make-win64-binaries.bat` batch script. 
 
 ## Initial Assumptions
 
  * `git` for Windows should be already installed in order to `git clone` the **osquery** repository containing the provisioning script.
- * The machine is running the Windows 10 64 bit operating system with PowerShell
+ * The machine is running the Windows 10 64 bit operating system with PowerShell 3.0 and later installed
```

Comment:
nit, word choice: The machine is running Windows 10 64-bit with PowerShell 3.0 or later